mne/gui/tests/test_ieeg_locate_gui.py

```diff
@@ -140,8 +140,11 @@ def test_ieeg_elec_locate_gui_display(_locate_ieeg, _fake_CT_coords):
 
     # test no seghead, fsaverage doesn't have seghead
     with pytest.warns(RuntimeWarning, match='`seghead` not found'):
-        _locate_ieeg(raw.info, trans, aligned_ct, subject='fsaverage',
-                     subjects_dir=subjects_dir)
+        with catch_logging() as log:
+            _locate_ieeg(raw.info, trans, aligned_ct, subject='fsaverage',
+                        subjects_dir=subjects_dir, verbose=True)
```

Comment:
@alexrockhill you just needed a `verbose=True` here, as you can't guarantee what the log level is on any given system (and we intentionally set the default to `WARNING` on at least some CIs to ensure we pass `verbose=True` when it's needed in tests)